Packs/PolySwarm/ReleaseNotes/2_0_4.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### PolySwarm
+
+- Added the *File.MD5*, *File.SHA1* and *File.SHA256* outputs for ***file*** command.
```

Comment:
They were already there, right? So you just added the listed outputs, but no functionality change.

Reply:
no, the opposite, i added the outputs, the functionality was already there

Reply:
Exactly, so the release notes should reflect this - you didn't add the outputs, but rather updated the reported outputs to the actual outputs.